packages/framework/src/Model/Article/ArticleDetailFriendlyUrlDataProvider.php

```diff
@@ -70,4 +74,22 @@ public function getRouteName(): string
     {
         return static::ROUTE_NAME;
     }
+
+    /**
+     * @param array $data
+     * @return \Shopsys\FrameworkBundle\Component\Router\FriendlyUrl\FriendlyUrlData
+     */
+    protected function createFriendlyUrlDataDeprecated(array $data): FriendlyUrlData
+    {
+        return $this->friendlyUrlDataFactory->createFromData($data);
+    }
+
+    /**
+     * @param array $data
+     * @return \Shopsys\FrameworkBundle\Component\Router\FriendlyUrl\FriendlyUrlData
+     */
+    protected function createFriendlyUrlDataNewWay(array $data): FriendlyUrlData
```

Comment:
what about `createFriendlyUrlDataNewWay(int $id, string $name): FriendlyUrlData` ?

Reply:
we discussed it - it is not important as the method will be removed in `8.0` (added @internal annotation to ensure that)